efi/preinstall/errors.go

```diff
@@ -139,6 +139,22 @@ func (e *joinError) Unwrap() []error {
 	return e.errs
 }
 
+// MissingKernelModuleError is returned unwrapped from [RunChecks] to indicate that
+// the specified kernel module is not built as part of the currently executing kernel,
+// but is required to be loaded in order for tests to continue. The caller is expected
+// to load the required kernel module.
+type MissingKernelModuleError string
+
+func (e MissingKernelModuleError) Error() string {
+	return fmt.Sprintf("the kernel module %q must be loaded", string(e))
+}
+
+// String implements [fmt.Stringer] and returns the name of the kernel module that
+// should be loaded before calling [RunChecks].
+func (e MissingKernelModuleError) String() string {
```

Comment:
I wonder if this should be Module instead?

Reply:
I did wonder about doing that. I think I'll keep the String method for formatting purposes but with no guarantees about the actual string value returned, and I'll add a Module method which returns just the module name.

Reply:
oh, actually, I'll just replace the String method with Module - I forgot that this was an error.